types/dynogels/dynogels-tests.ts

```diff
@@ -0,0 +1,795 @@
+import dynogels = require("dynogels");
```

Comment:
confusingly, it is fine to `import * as dynogels from 'dynogels'` since dynogels doesn't use the `export = ` syntax.

It works either way in this case, but note that you can `import { Model } from 'dynogels'`, which is not possible when importing from usages of the `export = ` syntax.

Reply:
sweet, thanks for the tips @sandersn !